package.json

```
@@ -23,9 +29,12 @@
     "eslint": "^7.32.0",
     "eslint-config-airbnb-base": "^14.2.1",
     "eslint-plugin-import": "^2.24.2",
+    "uuid": "^8.3.2",
     "jest": "^27.2.0"
   },
   "dependencies": {
-    "axios": "^0.21.4"
+    "axios": "^0.21.4",
+    "cpf-cnpj-validator": "^1.0.3",
+    "validator": "^13.6.0"
   }
-}
+}
```

Adicionar linha ao final

```suggestion
}

```